### PR TITLE
fix(client): server api version mismatch

### PR DESCRIPTION
**Pull request by liuxiaocs7**
## Purpose of the PR

https://github.com/hugegraph/hugegraph-test/pull/5#issuecomment-1833354718

api version is outdated in client will be handled in #546, this pr will handle the ci problem later~

## Verifying these changes

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [ ]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [x]  `Doc - No Need`

### CODE REVIEW

**Comment by liuxiaocs7** [comment]
> �thanks, could also update the commit in this PR image
> 
> maybe could choose `ed493f3093eab293a75cd2f539e29ca11fa6cd81` ([apache/incubator-hugegraph@ed493f3](https://github.com/apache/incubator-hugegraph/commit/ed493f3093eab293a75cd2f539e29ca11fa6cd81))

I see, thanks for the reply!~

File: hugegraph-client/assembly/travis/install-hugegraph-from-source.sh [hunk 1]
```diff
@@ -60,5 +61,5 @@ echo "gremlinserver.url=http://127.0.0.1:8282" >> ${REST_SERVER_CONFIG}
 
 # start HugeGraphServer with https protocol
 bin/init-store.sh
-bin/start-hugegraph.sh
+bin/start-hugegraph.sh -t 60
```
Comment (simon824) [inline hugegraph-client/assembly/travis/install-hugegraph-from-source.sh:64]: Server startup time seems to be extended
TODO：improve server start scripts later.

**Comment by simon824** [comment]
It seems to be caused by this [pr](https://github.com/apache/incubator-hugegraph/commit/2251adf073a10ef898caab04fc47df3b1dec2c5c#diff-5c82d8e7709b586124470a04ce5c8e56b3a59473b56e522788ac3482c423423fL144-R144) which removed the `-k` curl parameter.

**Comment by simon824** [comment]
refer to #550

**Pull request closed without merging**